src/OpenLoco/Audio/Audio.cpp

```diff
@@ -304,6 +305,7 @@ namespace OpenLoco::Audio
             }
             Mix_CloseAudio();
         }
+        alcOpenDevice(nullptr);
```

Comment:
This needs the corresponding `alcCloseDevice()` in `disposeDSound()`. I can see you aren't actually opening anything since nullptr is passed in, but it's probably safer to at least pencil in the call to `alcCloseDevice()` so we don't forget it later

Reply:
also do you think it's worth wrapping the openal calls into a simple object wrapper, so have a `class OpenAL` or similar that wraps all the functions we need. this would keep openal separate and self-contained, and make it very easy to unit test in future

Reply:
This was just to trigger the ci to see if i had got the dependencies setup correctly.